.github/workflows/uat-deploy-java-contracts.yml

```diff
@@ -28,4 +28,4 @@ jobs:
         run: |
           ./gradlew clean build 
           ./gradlew optimizedJar
-          ./gradlew deployToUat -PdeploymentENV=uat -PkeystoreName=$UAT_KEYSTORE_PATH -PkeystorePass=$UAT_PASSWD
+          ./gradlew xcall:deployToUat -PdeploymentENV=uat -PkeystoreName=$UAT_KEYSTORE_PATH -PkeystorePass=$UAT_PASSWD
```

Comment:
I believe 
`./gradlew deployToUat -PdeploymentENV=uat -PkeystoreName=$UAT_KEYSTORE_PATH -PkeystorePass=$UAT_PASSWD` command should suffice, considering that we also need to deploy the mock-dapp, which utilizes the xcall protocol.

Reply:
Corrected.